### PR TITLE
Fixing test3 to have enough tests to make sure we test all cases

### PR DESCRIPTION
**Pull request by calvinbrown085**
#434

### CODE REVIEW

File: src/verify.rs [hunk 1]
```diff
@@ -65,6 +65,7 @@ fn compile_and_run_interactively(exercise: &Exercise) -> Result<bool, ()> {
         Err(output) => {
             warn!("Ran {} with errors", exercise);
             println!("{}", output.stdout);
+            println!("{}", output.stderr);
```
Comment (jrvidal) [inline src/verify.rs:68]: I think this is a spurious change, isn't it?

**Review by jrvidal — APPROVED** [review]
LGTM :rocket:

File: src/verify.rs (hunk 1 above)
Comment (fmoko) [reply src/verify.rs:68]: yeah, i think this should go in a separate PR if anything

Comment (jrvidal) [reply src/verify.rs:68]: I think it's part of https://github.com/rust-lang/rustlings/pull/522/commits/9f61db5dbe38538cf06571fcdd5f806e7901e83a, which is already on `main`.

Comment (calvinbrown085) [reply src/verify.rs:68]: @jrvidal @fmoko That's my bad, I think I based my change off of master, instead of 4.0.0 which had another commit in it.

**Pull request merged**